### PR TITLE
Adding data-test attributes to ChatBot react component

### PR DESCRIPTION
**Pull request by elorzafe**
_Issue #, if available:_

_Description of changes:_
Adding `data-test` attributes to ChatBot on react component

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/aws-amplify-react/src/Amplify-UI/data-test-attributes.tsx [hunk 1]
```diff
@@ -143,6 +143,13 @@ export const genericAttrs = {
 	dialCodeSelect: 'dial-code-select',
 };
 
+export const chatBot = {
```
Comment (sammartinez) [inline packages/aws-amplify-react/src/Amplify-UI/data-test-attributes.tsx]: General question: Is Chat Bot only available in React? We may need to add these to Angular and Vue data test attrs files as well. Even if its a placeholder

File: packages/aws-amplify-react/src/Interactions/ChatBot.tsx [hunk 2]
```diff
@@ -441,7 +455,7 @@ export class ChatBot extends Component<IChatBotProps, IChatBotState> {
 						micButtonDisabled={this.state.micButtonDisabled}
 						handleMicButton={this.micButtonHandler}
 						currentVoiceState={this.state.currentVoiceState}
-					></ChatBotInputs>
+					/>
```
Comment (sammartinez) [inline packages/aws-amplify-react/src/Interactions/ChatBot.tsx]: Cleaner code!

File: packages/aws-amplify-react/src/Amplify-UI/data-test-attributes.tsx (hunk 1 above)
Comment (jordanranz) [reply packages/aws-amplify-react/src/Amplify-UI/data-test-attributes.tsx]: Yea, can we add the Angular and Vue data test attributes as well.

Comment (elorzafe) [reply packages/aws-amplify-react/src/Amplify-UI/data-test-attributes.tsx]: Cool, I can add those as well

**Pull request merged**